.github/PULL_REQUEST_TEMPLATE/accept-reject-pep.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Accept/Reject PEP
+about: Mark a draft PEP as accepted or rejected
+---
+
+<!--
+
+See PEP 1 for more details about the PEP review and resolution process:
+
+https://peps.python.org/pep-0001/#pep-review-resolution
+
+Make sure to include the PEP number in the pull request title; for example:
+
+PEP NNN: Mark as Accepted/Rejected
+
+Thanks!
+
+-->
+
+
+# Accept/Reject PEP
+
+<!--
+
+Link to the official acceptance/rejection message
+by the Steering Council/PEP Delegate on the PEP's discussion thread.
+
+-->
+
+
+
+## Checklist
+
+<!--
+
+This lists the key steps needed to mark a PEP as Accepted/Rejected.
```

Comment:
The link is in the proposed PEP text, so I don't think it *needs* to be in the PR message.

```suggestion

<!--

This checklist contains the key changes to mark a PEP as Accepted/Rejected.
```